db/seeds.rb

```diff
@@ -56,6 +56,18 @@
   role: :judge
 ).first_or_create!
 
+first_task_criterium_1 = Repo::TaskCriterium.where(
+  title: "First criterion",
+  max_value: 10,
+  task:   first_task
+).first_or_create!
+
+second_task_criterium_1 = Repo::TaskCriterium.where(
+  title: "Second criterion",
+  max_value: 5,
+  task:   first_task
```

Comment:
hm, I am not sure about the business logic of it, but is it possible for tasks to have more than one criterium?
@sveredyuk could you clarify this for us?

Reply:
Yes. task has many criteria: Code Quality, API design, Tests, etc...